src/screens/AddVideo/index.tsx

```diff
@@ -1,16 +1,30 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import AddVideoComponent, { Level } from './AddVideo.component';
 import { youTubeConfig } from '../../config';
 import { AddVideoFormUrl } from './AddVideoFormik';
 import { useSelector } from 'react-redux';
 import { userSelectors } from '../../redux/user';
+import fireApi from '../../services/firebase';
 
 const AddVideo = () => {
   const [videoObject, setVideoObject] = useState<Object>({});
+  const [videoIsDuplicate, setVideoIsDuplicate] = useState<boolean>(false);
   const userId = useSelector(userSelectors.userId);
   const axios = require('axios');
   const getVideoId = require('get-video-id');
 
+  useEffect(() => {
+    fireApi.addToDb('videos', videoObject);
+  }, [videoObject]);
+
+  const setVideoIsDuplicateToTrue = () => {
+    setVideoIsDuplicate(true);
+  };
+
+  const setVideoIsDuplicateToFalse = () => {
+    setVideoIsDuplicate(false);
+  };
+
   const youTubeApiCall = (
```

Comment:
i'd move this function to new youTubeService (like firebase)

Reply:
i'd create dedicated function in fireApi for adding video, so it can be typed properly